python/lsst/ts/ATDome/dome_csc.py

```diff
@@ -130,44 +149,58 @@ def __init__(self, config_dir=None, initial_state=salobj.State.STANDBY,
             raise salobj.ExpectedError("Cannot move azimuth while homing")
         azimuth = data.azimuth
         if azimuth < 0 or azimuth > 360:
```

Comment:
What happens when azimuth = 360 = 0? are both value allowed and it phase wraps?

Reply:
Both are allowed because it turns out to be basically impossible to make an Angle class (one that understands different units, such as in astropy) that can reliably fill the range [0, 360) deg, due to roundoff error.

They both send the dome to the same place in any case.

That said, if the real dome rejects 360 then I should change this to >= 360. Can you test that for me?

Reply:
![image](https://user-images.githubusercontent.com/14132079/74056204-c8c16780-499e-11ea-9f49-2e6fc1fc5f1c.png)

Reply:
Thank you very much.